### ISSUE TITLE
[EPIC] Import of external faults and fractures

### ISSUE THREAD

**Issue by CusiniM** [GEOSX/GEOS#2370]
## Description
The goal of this issue is to track progress on import of external faults and fractures through VTK, both for conforming and embedded discretization approaches.

## Work Scope
- import of conforming features (faults and fractures) through vtk
- import of embedded fracture networks through vtk

## Issues in this EPIC
- [ ] https://github.com/GEOSX/GEOS/issues/2308
- [ ] Placeholder for issue/PR for import of embedded discrete fracture networks from an external format

**Comment by untereiner** [comment]
I am looking on a way to reuse the output of GEOS as an input. I stumbled on this article [talking about deprecation of multiblocks](https://discourse.paraview.org/t/deprecating-multiblock-datasets/4950).  I think Faults and Fractures would benefit from the vtkPartitionedDataSetCollection object. The fractures could be organized with a vtkDataAssembly (organized also with the CellElementRegions). What do you think ?

**Comment by TotoGaz** [comment]
Hello @untereiner 

We had a talk with kitware and `vtkMultiBlockDataSet` is not deprecated.
We also reached the conclusion that it was better for us to consistently use `vtkMultiBlockDataSet` in `geos` and maybe migrate if we need all at once.

In the way to reusing the outputs, one question was about the nodes.
We currently write each region in its own block; so nodes used for the definitions of multiple regions will be defined in multiple blocks. IIRC our discussion with kitware, that would force us to manage the "merge". Nothing that can't be done, but still to be done.

Eventually, it's OK to support `vtkPartitionedDataSet*` utilities if this brings us some new cool features 😉

**Comment by untereiner** [comment]
Yes, well, it is too much used to deprecate it... but a replacement is already there...
I manually transformed a vtkMultiBockDataSet to a vtkPartitionedDataSet with paraview. Yes you would have nodes as the leafs of the tree and for each node multiple regions. It is needs special care.
For what I heard, some users would find reusing outputs as a new cool feature :smiley:
